### PR TITLE
Updated doc for tf.keras.backend.maximum

### PR DESCRIPTION
**Pull request by lukasfolle**
I added some information concerning the returns and a basic example.
Any improvments or ideas on that?

### CODE REVIEW

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -277,6 +277,9 @@ def linear(x):
 
   Returns:
       The linear activation: `x`.
+
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: I don't think this is necessary. Not many use linear at regression output.

File: tensorflow/python/keras/backend.py [hunk 2]
```diff
@@ -2245,7 +2245,19 @@ def maximum(x, y):
       y: Tensor or variable.
 
   Returns:
-      A tensor.
+      A tensor with the maximum value(s) of `x` and `y.
```
Comment (tanzhenyu) [inline tensorflow/python/keras/backend.py]: ... element wise.

Comment (lufol) [reply tensorflow/python/keras/backend.py]: Done in 7922571

File: tensorflow/python/keras/backend.py [hunk 3]
```diff
@@ -2245,7 +2245,19 @@ def maximum(x, y):
       y: Tensor or variable.
 
   Returns:
-      A tensor.
+      A tensor with the element wise maximum value(s) of `x` and `y.
```
Comment (tanzhenyu) [inline tensorflow/python/keras/backend.py]: typo, `y`.

Comment (lufol) [reply tensorflow/python/keras/backend.py]: Thanks, fixed in 5cd0705.

**Comment by gbaned** [comment]
@lufol Could you please resolve the conflicts? Thanks!

**Comment by lufol** [comment]
@gbaned Done ✅

**Review by tanzhenyu — APPROVED** [review]

**Pull request merged**